### ISSUE TITLE
A proper README

### ISSUE THREAD

**Issue by olivierlambert** [vatesfr/pulumi-xenorchestra#2]
We should have a nice `README.me` and probably also ideally a `MAINTAINERS.md`. When it's done, I'll do the publicity around it and we'll see if we could have even more contributors :+1:

**Comment by belfhi** [comment]
I agree, but in order explain how to use this provider it also needs to be published in various package registries.

**Comment by gCyrille** [comment]
Closed by #340
